src/app/component/ProductReviewRating/ProductReviewRating.component.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './ProductReviewRating.style';
+
+/**
+ * @class ProductReviewRating
+ */
+class ProductReviewRating extends Component {
+    render() {
+        const { summary } = this.props;
+        const clip = `polygon(0 0, 0 100%, ${summary}% 100%, ${summary}% 0)`;
+
+        return (
+            <div block="ProductReviewRating" title={ `${summary}%` }>
+                <div block="ProductReviewRating" elem="RatingStars">
+                    <i className="star" />
```

Comment:
What meaning does `<i>` contains? Have you seen a video I shared? This is wrong implementation of rating.

Reply:
I have watched the video you share and thought that is is meant more for the from rating inputs, was not sure about the static rating stars. Could you please further explain (maybe in Slack) the static rating implementation that would be preferred?